pkg/controllers/disruption/drift.go

```diff
@@ -101,9 +99,7 @@ func (d *Drift) ComputeCommand(ctx context.Context, candidates ...*Candidate) (C
 			}
 			return Command{}, err
 		}
-		// Log when all pods can't schedule, as the command will get executed immediately.
 		if !results.AllNonPendingPodsScheduled() {
-			logging.FromContext(ctx).With(lo.Ternary(candidate.NodeClaim.IsMachine, "machine", "nodeclaim"), candidate.NodeClaim.Name, "node", candidate.Node.Name).Debugf("cannot terminate since scheduling simulation failed to schedule all pods %s", results.NonPendingPodSchedulingErrors())
```

Comment:
Do we replace this log elsewhere?

Reply:
Nope, we had this in here when Karpenter would choose to expire/drift a node without needing all pods to reschedule. Since this doesn't actually result in any action, I think this is better driven by node events, rather than red herring logs.

Reply:
I'd love to drive each of our high level controller actions with each of:
1. Log line
2. Event
3. Metric

Reply:
Does each controller action or in-action require a log line? This one in particular I don't think requires a log line. Are we agreed on that?